CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `get_taxon` to methods
 - Added `streptococcus` and `streptococcus_pyogenes` workflows and profiles to configs
 - Added ska filepath to yaml
+- Added optional read downsampling using seqtk.
```

Comment:
We're not adding the removal of human reads right?

Reply:
Not in this one. Better to do feature by feature so we can ensure that everything works